riak/transports/pbc/connection.py

```diff
@@ -165,7 +165,7 @@ def _ssl_handshake(self):
 
                     return True
                 except ssl.SSLError as e:
-                    raise SecurityError(e.library + ": " + e.reason)
```

Comment:
`library` was not an attribute of `SSLError` in my testing. Just wrapping the exception provides more information to the user.